equinix/data_source_ecx_l2_sellerprofiles.go

```diff
@@ -111,10 +112,10 @@ func dataSourceECXL2SellerProfilesRead(ctx context.Context, d *schema.ResourceDa
 				continue
 			}
 		}
-		if len(metros) > 0 && !atLeastOneStringFound(metros, flattenECXL2SellerProfileMetroCodes(profile.Metros)) {
+		if len(metros) > 0 && !slices.ContainsOne(metros, flattenECXL2SellerProfileMetroCodes(profile.Metros)) {
```

Comment:
Everything affected by the move of the private string slice functions is either a NE or an ECXv3 resource.  Might be preferable (lower risk, simpler diff) to move the custom implementations of these functions to a package in `internal` rather than replacing them with `slices` functionality.

Reply:
The latest version of this PR attempts to make the code changes as minimal as possible; rather than moving things to separate packages or replacing internal functions with "better" alternatives, the updated PR just does whatever is needed to get us to the point of isolating the SDK provider code without changing any existing logic.